.eslintignore

```diff
@@ -1,4 +1,5 @@
 webpack/*
 karma.conf.js
 tests.webpack.js
-dist
\ No newline at end of file
+dist
+/**/*.d.ts
```

Comment:
Is there any reason we're ignoring these definition files and have used the `.d.ts` extension instead of `.types.ts`?

Reply:
Sorry, this is from previous tries and should be removed

Reply:
Ah okay no worries.